src/components/VoteButton.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+import { voteReview } from '../api.js'
+
+export function VoteButton({ reviewID, setReview, setDisplayAlert }) {
+  function vote(incVote) {
+    const req = { inc_votes: incVote }
+    setReview((review) => {
+      // optimistic rendering
+      const updatedReview = { ...review }
+      updatedReview.votes += incVote
+      return updatedReview
```

Comment:
Yes, good to see you creating a copy of the review, to avoid mutating state, and then returning the new object, well done!